core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/changelog/BoxChangeLog.java

```diff
@@ -0,0 +1,44 @@
+package uk.ac.bbsrc.tgac.miso.core.data.impl.changelog;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+import org.hibernate.annotations.Index;
+import org.hibernate.annotations.Table;
+
+import uk.ac.bbsrc.tgac.miso.core.data.AbstractChangeLog;
+import uk.ac.bbsrc.tgac.miso.core.data.Box;
+import uk.ac.bbsrc.tgac.miso.core.data.impl.BoxImpl;
+
+@Entity
+@Table(appliesTo = "BoxChangeLog", indexes = { @Index(name = "BoxChangeLog_boxId_changeTime", columnNames = { "boxId", "changeTime" }) })
+public class BoxChangeLog extends AbstractChangeLog {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long boxChangeLogId;
```

Comment:
Hibernate classes require an id. I decided that the easiest way to satisfy this requirement was to add PK fields to the change log tables.